priv/templates/talon.gen.theme/admin_lte/templates/layout/sidebar.html.slim

```diff
@@ -1,11 +1,19 @@
 aside.main-sidebar
   section.sidebar style="height: auto;"
     ul.sidebar-menu
+      li.header.text-uppercase Dashboard
+      / TODO: use talon_context to get page/dashboard_paths (DJS)
+      - resource_or_page = @talon_resource || @talon_page
+      = if resource_or_page do
+        = for dashboard_path <- resource_or_page.dashboard_paths(@conn.assigns.talon) do
```

Comment:
We should have a UI discussion for this. I'm thinking the UI should be:

Pages
  Dashboard
  MyPage1
  ...
Resources
  Users
  Posts
  ...
Actions
  ...

And therefore, we should be calling a `pages_paths` function to return the collection of pages.